pkg/gardenadm/cmd/bootstrap/bootstrap.go

```diff
@@ -208,8 +224,66 @@ func run(ctx context.Context, opts *Options) error {
 		})
 		// TODO(timebertt): destroy Bastion after successfully bootstrapping the control plane
 
-		_ = waitUntilWorkerReady
+		// Delete machine-controller-manager to prevent it from interfering with Machine objects that will be migrated to
+		// the autonomous shoot.
+		deleteMachineControllerManager = g.Add(flow.Task{
+			Name:         "Deleting machine-controller-manager",
+			Fn:           component.OpDestroyAndWait(b.Shoot.Components.ControlPlane.MachineControllerManager).Deploy,
```

Comment:
```suggestion
			Fn:           component.OpDestroyAndWait(b.Shoot.Components.ControlPlane.MachineControllerManager).Destroy,
```

Just to prevent confusion :)